### PR TITLE
Add display all variables if true variable 

### PR DESCRIPTION
**Pull request by patsevanton**
display_all_variables by default
Add examples to inventory

### CODE REVIEW

File: contrib/ansible/roles/skydive_common/tasks/main.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 - name: Display all variables/facts known for a host
   debug:
     var: hostvars[inventory_hostname]
+  when: display_all_variables|default(false)|bool == true
```
Comment (safchain) [inline contrib/ansible/roles/skydive_common/tasks/main.yml]: I would prefer to use `verbosity`. it seems to handle what you try to introduce nicely. What do you think ?
https://docs.ansible.com/ansible/latest/modules/debug_module.html

Comment (patsevanton) [reply contrib/ansible/roles/skydive_common/tasks/main.yml]:
```- name: Display all variables/facts known for a host
  debug:
    var: hostvars[inventory_hostname]
    verbosity: 4```

Is it ok?

Comment (safchain) [reply contrib/ansible/roles/skydive_common/tasks/main.yml]: yes

**Comment by patsevanton** [comment]
For Clean history
https://github.com/skydive-project/skydive/pull/2106

**Pull request closed without merging**